exercises/pov/Example.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class Graph<T> : IEquatable<Graph<T>>
+{
+    public Graph(T value, IEnumerable<Graph<T>> children)
+    {
+        Value = value;
+        Children = children;
+    }
+
+    public T Value { get; }
+    public IEnumerable<Graph<T>> Children { get; }
+
+    public bool Equals(Graph<T> other) =>
+        Value.Equals(other.Value) && Children.SequenceEqual(other.Children);
+}
+
+public class Crumb<T>
+{
+    public Crumb(T value, IEnumerable<Graph<T>> left, IEnumerable<Graph<T>> right)
+    {
+        Value = value;
+        Left = left;
+        Right = right;
+    }
+
+    public T Value { get; }
+    public IEnumerable<Graph<T>> Left { get; }
+    public IEnumerable<Graph<T>> Right { get; }
+}
+
+public class Zipper<T>
+{
+    public Zipper(Graph<T> focus, IEnumerable<Crumb<T>> crumbs)
+    {
+        Focus = focus;
+        Crumbs = crumbs;
+    }
+
+    public Graph<T> Focus { get; }
+    public IEnumerable<Crumb<T>> Crumbs { get; }
+}
+
+public static class Pov
+{
+    public static Graph<T> CreateGraph<T>(T value, IEnumerable<Graph<T>> children) 
+        => new Graph<T>(value, children);
+
+    public static Graph<T> FromPOV<T>(T value, Graph<T> graph) where T : IComparable 
```

Comment:
This is minor, but based on [C# style guidelines](https://msdn.microsoft.com/en-us/library/x2dbyw72(v=vs.71).aspx) this should be `FromPov` since POV is more than two letters. That would also make it consistent with the class name.